include/lug/Main/MainAndroid.hpp

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <android/native_activity.h>
+#include <android/configuration.h>
+#include <thread>
```

Comment:
Don't include thread if you aren't using it in the header! (Include it in the source file)